### PR TITLE
Feature/add ansi color into menu title

### LINKED ISSUE #11107: [OSX] Support ANSI color in the Tray menubar
* Electron version: all
* Operating system: OSX

### Expected behavior
The `setTitle()` method can support the ANSI colors

### Actual behavior
The `setTitle()` method do not support the ANSI colors

### PR DESCRIPTION
**Pull request by oktapodia**
Fix #11107 

I did not add a test, this is my first objC PR and I have not seen any test about the tray menu

`\033[34mI \033[32mam \033[31ma \033[33mtest \033[0m` will be rr

### CODE REVIEW

**Comment by oktapodia** [comment]
I don't know/understand how to resolve the lint errors to be honest

File: atom/browser/ui/cocoa/NSColor+Hex.h [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#import <string>
+
+@interface NSColor(Hex)
+
++ (NSColor*) colorWithWebColorString:(NSString*)color;
++ (NSColor*) colorWithHexColorString:(NSString*)hex;
+
+@end
+
+#endif  // ATOM_BROWSER_UI_COCOA_NSCOLOR_HEX_H_
```
Comment (TiagoDanin) [inline atom/browser/ui/cocoa/NSColor+Hex.h]: Two space here
https://travis-ci.org/electron/electron/jobs/301514595#L736

**Comment by oktapodia** [comment]
Yes @TiagoDanin, I saw that, but can you give me an example, please? I tried many things, the linter was okay, but it was not possible to build the app anymore...

File: atom/browser/ui/cocoa/NSColor+Hex.h (hunk 1 above)
Comment (oktapodia) [reply atom/browser/ui/cocoa/NSColor+Hex.h]: Fixed

**Comment by TiagoDanin** [comment]
@oktapodia I recommend you see this question on Stackoverflow
https://stackoverflow.com/questions/40866352/how-to-do-a-reinterpret-cast-with-obj-c

**Pull request merged**